assignments/objects.js

```diff
@@ -49,16 +67,38 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  name: 'Susan',
```

Comment:
Good job here.  This is going to become very useful when you get to backend working with databases.